frontend/src/Parse/ParseResult.tsx

```diff
@@ -228,7 +228,13 @@ function ParseResult(props: ParseResultProps) {
 
         <ParseResultItem
           title={translate('CustomFormats')}
-          data={<EpisodeFormats formats={customFormats} />}
+          data={
+            customFormats && customFormats.length > 0 ? (
```

Comment:
```suggestion
            !!customFormats?.length ? (
```

or keep the `> 0` 🤷

Reply:
lint doesn't like this:
```
  232:13  warning  Unexpected negated condition  no-negated-condition
  232:13  error    Redundant double negation     no-extra-boolean-cast
```

Reply:
Ahh right, `customFormats?.length` should be sufficient

Reply:
yep